charts/camunda-platform/charts/zeebe-gateway/templates/gateway-deployment.yaml

```diff
@@ -139,6 +139,9 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           {{- end }}
+      {{- if .Values.sidecars}}
```

Comment:
A space is missing between the key and brackets.

Reply:
I'll fix this.